test/unit/carousel.ts

```diff
@@ -0,0 +1,545 @@
+import { utils } from '@storefront/core';
+import Carousel, { calSwipeDirection } from '../../src/carousel';
+import suite from './_suite';
+
+suite('Carousel', ({ expect, spy, stub }) => {
+  let carousel: Carousel;
+  let DEFAULT_SETTINGS;
+
+  describe('settings', () => {
+    before(() => {
```

Comment:
Can move this to the top and get rid of the beforeEach below. Also should be `beforeEach`